rpm/xulrunner-qt5.spec

```diff
@@ -356,13 +418,13 @@ echo "ac_add_options --disable-elf-hack" >> "$MOZCONFIG"
 # but as sailfish-browser has privileged EGID, glibc removes it for security reasons. 
 # Set ELF RPATH through LDFLAGS. Needed for plugin-container and libxul.so
 # Additionally we limit the memory usage during linking
- echo 'FIX_LDFLAGS="-Wl,--reduce-memory-overheads -Wl,--no-keep-memory -Wl,-rpath=%{mozappdir}"' >> "${MOZCONFIG}"
+ echo 'FIX_LDFLAGS="-Wl,--strip-debug -Wl,--gc-sections -Wl,--reduce-memory-overheads -Wl,--no-keep-memory -Wl,-rpath=%{mozappdir}"' >> "${MOZCONFIG}"
  echo 'export LDFLAGS="$FIX_LDFLAGS"' >> "${MOZCONFIG}"
  echo 'LDFLAGS="$FIX_LDFLAGS"' >> "${MOZCONFIG}"
  echo 'export WRAP_LDFLAGS="$FIX_LDFLAGS"' >> "${MOZCONFIG}"
  echo 'mk_add_options LDFLAGS="$FIX_LDFLAGS"' >> "${MOZCONFIG}"
 
-RPM_BUILD_NCPUS=`nproc`
+RPM_BUILD_NCPUS=4
```

Comment:
Not sure if that is still needed anymore? Was this a debug thing?

Reply:
I've switched this back to the original now.